scripts/package-images

```diff
@@ -8,4 +8,4 @@ source ./scripts/version.sh
 mkdir -p dist/artifacts
 
 cp -f build/images.txt dist/artifacts/${PROG}-images.${PLATFORM}.txt
-gzip < build/images/${PROG}-airgap.tar > dist/artifacts/${PROG}-images.${PLATFORM}.tar.gz
+gzip < build/${PROG}-airgap.tar > dist/artifacts/${PROG}-images.${PLATFORM}.tar.gz
```

Comment:
`build/images` is mounted into dev-shell at `/var/lib/rancher/rke2/agent/images`. This change will have the side-effect of forcing rke2 to reach out to internet registries for all images other than `rancher/rke2-runtime` and `rancher/hardened-kubernetes`. Is that intentional?

Reply:
The problem was that I couldn't get the air-gap tar file into the test image, presumably because `build/images` is in `.dockerignore`. Am thinking either we should remove that entry from `.dockerignore` or change dev-shell to use `build`.